src/instruments/instruments.es6

```diff
@@ -8,57 +8,62 @@ import "common/util";
 
 function refresh_active_symbols() {
     liveapi
-        .cached
-        .send({ active_symbols: 'brief' })
-        .then(function(data) {
-            local_storage.set('active_symbols', data.active_symbols);
-            const active_symbols = [];
-            const active_markets = _(data.active_symbols).groupBy('market').map(function(symbols) {
-             
-                const filtered_symbols = filterRestrictedSymbols(symbols)
-  
-                const sym = _.head(filtered_symbols);
-     
-                const market = { name: sym.market, display_name: sym.market_display_name };
-                market.submarkets = _(filtered_symbols).groupBy('submarket').map(function(symbols) {
-                    const sym = _.head(symbols);
-                    const submarket = { name: sym.submarket, display_name: sym.submarket_display_name };
-                    submarket.instruments = _.map(symbols, function(sym) {
-                        active_symbols.push(sym.symbol);
-                        return {
-                            symbol: sym.symbol,
-                            display_name: sym.display_name,
-                        };
-                    });
-                    return submarket;
+      .cached
+      .authorize()
```

Comment:
Shouldn't we check for logged in state before waiting on authorize? Since this requires a token to authorize a user and guest users don't have this. https://developers.binary.com/api/#authorize

Reply:
yeah, you're right, will be fixed